lib/rubocop/cops_documentation_generator.rb

```diff
@@ -181,13 +181,13 @@ def code_example(ruby_code)
   end
 
   def configurations(department, pars, cop)
-    return '' if pars.empty?
```

Comment:
I also have a feeling that `pars` is a typo for `pairs` here.

Reply:
The variable `pars` appears multiple times in this file, so it doesn't seem to be a simple typo.
I'm not sure about the origin of this variable name, but it might possibly be short for `parameter`.

If we plan to refactor the variable name, it would be better to do so in a separate pull request.

Reply:
Yeah, that's fair. Even if it stands for parameters I think `params` reads a lot better. I'm also wondering about the ordering of the params - `department, pars, cop`, as something like `department, cop, pars` seems more natural to me. At any rate, that's outside the scope of your PR.

Reply:
I fixed it in #13713 :wink: